test/distributed/fsdp/test_fsdp_clip_grad_norm.py

```diff
@@ -209,6 +209,34 @@ def _test_ddp_parity(
                 self.assertEqual(n1, n2)
                 self.assertEqual(p1, p2)
 
+        if offload_params:
+            # TODO: Gradient computation on CPU and GPU differ slightly causing
+            # drift unrelated to `clip_grad_norm_()`.
+            # https://github.com/pytorch/pytorch/issues/89133
+            return
```

Comment:
A bit hacky, but would it help if we just ran DDP on CPU for this case if having this coverage is important?

Reply:
Does DDP work on CPU with NCCL backend?